src/Elcodi/Bundle/GeoBundle/Resources/config/validation.yml

```diff
@@ -0,0 +1,4 @@
+Elcodi\Component\Geo\Entity\Address:
+    properties:
+        city:
+            - Elcodi\Component\Geo\Validator\Constraints\CityExists: ~
```

Comment:
Nope, city_exists is the tag for the constraint validator, and we need here the constraint :)